docs/description/description.json

```diff
@@ -36,227 +36,37 @@
 }, {
   "patternId" : "B108",
   "title" : "hardcoded_tmp_directory",
-  "description" : "B108: Test for insecure usage of tmp file/directory",
+  "description" : "Buffer()",
   "parameters" : [ ]
 }, {
   "patternId" : "B109",
   "title" : "Test for a password based config option not marked secret",
-  "description" : "B109: Test for a password based config option not marked secret",
+  "description" : "Buffer()",
   "parameters" : [ ]
 }, {
   "patternId" : "B110",
   "title" : "try_except_pass",
-  "description" : "B110: Test for a pass in the except block",
+  "description" : "Buffer()",
   "parameters" : [ ]
 }, {
   "patternId" : "B111",
   "title" : "Test for the use of rootwrap running as root",
-  "description" : "B111: Test for the use of rootwrap running as root",
+  "description" : "Buffer()",
   "parameters" : [ ]
 }, {
   "patternId" : "B112",
   "title" : "try_except_continue",
-  "description" : "B112: Test for a continue in the except block",
+  "description" : "Buffer()",
   "parameters" : [ ]
 }, {
   "patternId" : "B201",
   "title" : "flask_debug_true",
-  "description" : "B201: Test for use of flask app with debug set to true",
-  "parameters" : [ ]
-}, {
```

Comment:
Did they delete all these patterns? Can you check?
If not, we might need to update the documentation generator to handle their changes

Reply:
Yes, it looks like they deleted these patterns: https://github.com/PyCQA/bandit/tree/main/doc/source/plugins